Core/Utilities/MySQL.py

```diff
@@ -458,7 +458,7 @@ def __escapeString( self, myString ):
     It also includes quotation marks " around the given string
     """
 
-    retDict = self.__getConnection()
+    retDict = self._getConnection()
```

Comment:
Why Do you changed this method from private to protected? In low level private methods are very useful.